frontend/app/forms/petition-for-dismissal-or-acquittal/petition-for-dismissal-or-acquittal.web.component.js

```diff
@@ -17,6 +28,110 @@ export default function PetitionForDismissalOrAcquittal_Web(props) {
         <TextInput dataKey="person.homePhone" label={__("home phone")} />
         <TextInput dataKey="person.email" label={__("email address")} />
       </Section>
+      <Section name="2. Case Information">
+        <Select
+          dataKey="person.petitionerRepresentative"
+          label={__("are you filling this out for yourself")}
+          options={petitionerRepresentativeOptions}
+        />
+        {data.person.petitionerRepresentative === "attorney" && (
+          <TextInput
+            dataKey="person.petitionerBarNumber"
+            label="What is your Utah Bar number?"
+          />
+        )}
+        <Radio
+          label="Court Type"
+          dataKey="case.courtType"
+          options={courtTypeOptions}
+        />
+        {data.case.courtType === "District" && (
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="District Courts"
+            groupOptions={DistrictCourtList}
+          />
+        )}
+        {data.case.courtType === "Justice" && (
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="Justice Courts"
+            groupOptions={JusticeCourtList}
+          />
+        )}
+        <TextInput dataKey="case.caseNumber" label={__("case number")} />
+        <TextInput dataKey="case.judgeName" label={__("judge full name")} />
+        <TextInput dataKey="case.arrestedDate" label={__("arrested date")} />
+        <TextInput
+          dataKey="case.lawEnforcementAgencysFileNumber"
+          label={__("the law enforcement agency's file number")}
+        />
+        <TextInput
+          dataKey="case.resultOfArrest"
+          label={__(
+            "the following court case was filed as a result of the arrest"
+          )}
+        />
+      </Section>
+      <Section name="3. Choose all that applies">
+        <Checkbox
+          dataKey="case.wasNotConvicted"
```

Comment:
This and the next few dataKeys are brand new data keys, right? I think that makes sense -- i don't think other forms ask these questions.

Reply:
Yes, they are new. I don't think any of the forms currently use those questions, but I believe the [petition for charges never filed](https://github.com/JustUtahCoders/utahexpungements.org/issues/30) will share some of those data keys.